src/signer_node/mod.rs

```diff
@@ -173,7 +177,7 @@ impl<T: TapyrusApi, C: ConnectionManager> SignerNode<T, C> {
 
         // Start First Round
         log::info!("Start block creation rounds.");
-        self.current_state = self.start_next_round();
+        self.start_next_round(0);
```

Comment:
You can define 0 as a constant like `NodeState::InitialMasterIndex = 0`?

Reply:
thanks I'll change it.